dev/docker/trino/Dockerfile

```diff
@@ -45,11 +45,18 @@ RUN rm -rf /usr/lib/trino/plugin/accumulo  \
     && rm -rf /usr/lib/trino/plugin/redis \
     && rm -rf /usr/lib/trino/plugin/session-property-managers \
     && rm -rf /usr/lib/trino/plugin/teradata-functions
-
 COPY --chown=trino:trino packages/trino/conf /etc/trino
 COPY --chown=trino:trino packages/gravitino-trino-connector/mysql-connector-java-8.0.27.jar /usr/lib/trino/plugin/iceberg/
 COPY --chown=trino:trino packages/gravitino-trino-connector/postgresql-42.7.0.jar /usr/lib/trino/plugin/iceberg/
-COPY --chown=trino:trino packages/gravitino-trino-connector /usr/lib/trino/plugin/gravitino
+
+ARG IMAGE_NAME
+RUN if [ "$IMAGE_NAME" = "datastrato/trino" ] ; then echo "Copying files for ${IMAGE_NAME} image" \
+  && COPY --chown=trino:trino packages/gravitino-trino-connector/mysql-connector-java-8.0.27.jar /usr/lib/trino/plugin/gravitino/ \
```

Comment:
For the user image, we should include the gravitino trinio connector.

Reply:
I have noticed this and I am considering how to address it.